dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java

```diff
@@ -90,6 +90,13 @@ public void testInitAutoMLTask() throws Exception {
         String command = mlflowTask.buildCommand();
     }
 
+    @Test
+    public void testInitCustomProjectTask() throws Exception {
+        MlflowTask mlflowTask = initTask(createCustomProjectParameters());
+        String command = mlflowTask.buildCommand();
+        mlflowTask.handle();
```

Comment:
Assert should be used in some of tests to verify.

Reply:
Thanks for your suggestions. I have fixed these.